templates/script/script.js

```diff
@@ -1,153 +1,133 @@
-            document.getElementById('name_error').style.visibility='visible';
-            document.getElementById("name").focus(); 
-        }
-
+    else {
+            RaizeError(window.event.target.name, 0);
     }
 }
-function checkEmail()
-{
-    var email_str = document.getElementById("email").value;
-    var email_reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,4})+$/;
-    document.getElementById('email_error').style.visibility='hidden';
-    document.getElementById('email_error').innerHTML=null;
-    if(email_str.length == 0)
-    {
-        document.getElementById('email_error').innerHTML="Please enter valid email";
-        document.getElementById('email_error').style.visibility='visible';
-        document.getElementById("email").focus();
+
+function checkEmail() {
+    let emailReg = new RegExp(/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,4})+$/);
+
+    if ((elEmail.value.length === 0) || emailReg.test(elEmail.value) !== true) {
+        RaizeError(window.event.target.name, 1);
     }
-    else
-    {
-        email_str = email_reg.test(email_str);
-        if(email_str != true)
-        {
-            document.getElementById('email_error').innerHTML="Please enter valid email";
-            document.getElementById('email_error').style.visibility='visible';
-            document.getElementById("email").focus(); 
-        }
+    else {
+            RaizeError(window.event.target.name, 0);
     }
 }
 
 function checkPhone()
 {
-    var phone_str = document.getElementById("phone").value;
-    var phone_reg = /^\+375(25|29|33|44)\d{7}$/g;
-    var phone_reg1 = /^\8017(2|3)\d{6}$/g;
-    document.getElementById('phone_error').style.visibility='hidden';
-    document.getElementById('phone_error').innerHTML=null;
-    var phone_result;
-    if(phone.length == 0)
-    {
-        document.getElementById('phone_error').innerHTML="Please enter valid phone number";
-        document.getElementById('phone_error').style.visibility='visible';
-        document.getElementById("phone").focus();
+    let phoneReg = new RegExp(/^\+375(25|29|33|44)\d{7}$/g);
+    let phoneReg1 = new RegExp(/^\8017(2|3)\d{6}$/g);
+
+    if ((elPhone.value.length === 0) || (phoneReg.test(elPhone.value) !== true && phoneReg1.test(elPhone.value) !== true)) {
+        RaizeError(window.event.target.name, 1);
+    }
+    else {
+            RaizeError(window.event.target.name, 0);
     }
-    else
-    {
-        phone_result = phone_reg.test(phone_str);
-        if(phone_result != true)
-        {
-            phone_result = phone_reg1.test(phone_str);
-            if(phone_result != true)
-            {
-                document.getElementById('phone_error').innerHTML="Please enter valid phone number";
-                document.getElementById('phone_error').style.visibility='visible';
-                document.getElementById("phone").focus();
-            } 
+}
+
+function RaizeError(element, on) {
```

Comment:
Please rename the function to reflect what it really does, cause right now I'd expect it to throw an error